### DISCUSSION TITLE
FYI Atlas Scientific Industrial Gen 3 Probes with SMA Connectors

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#906] [General]
Folks,

I want to let you know that it is possible to get the probes with SMA connectors even if they are not shown as an option in the pull down menu.  For the past couple of weeks, the only option from the pull down menu on their web site is "tinned lead."  

Last week I sent an email to their customer service inquiring when the SMA connector version of the probes would be available.  I received a response within a few minutes saying I could request SMA connectors in the "Notes" section of the order form and they would make the probes with the SMA connectors for me.  I was a little skeptical but I went ahead & ordered a pH probe and an ORP probe & requested the SMA connector version.  When I received the order number, I sent a note to customer service & provided the order number & support request number.  A little while later, I received a note from customer service saying they were making the probes as I requested.

A few minutes ago the probes arrived & they had the SMA connectors installed.  To say the least, I was delighted.

If you need to get these probes with SMA connectors &  it's not shown as an option, I suggest you write to customer support & ask for their help.  *So* much easier than dealing with installing them yourself!!!

Now if I can just get the bloody tape off of the probes...  As described in the Wiki, it may be the hardest part of the process.  Suggestions for that challenge are most welcome!!

**Comment by rstrouse** [comment]
That tape is a menace.  I had nightmares about being taped to cement shoes with that stuff then going for a swim.

**Reply by FlaMike** [reply]
Hope you had a *very* long snorkel or huge air tanks!  Watta nightmare.  There must be a better way to assure a watertight seal for shipping & storage other than the tape from hell.
